copier-answers.jso.yml

```diff
@@ -0,0 +1,8 @@
+PackageName: "KnetNLPModels"
+PackageOwner: "JuliaSmoothOptimizers"
+PackageUUID: "9601d21a-7b1e-4fc7-9dab-5c3f545997d5"
+_src_path: "https://github.com/JuliaSmoothOptimizers/JSOBestieTemplate.jl"
```

Comment:
this is a private repo, would that be a problem?

Reply:
@tmigot this is the only concern I have

Reply:
I will make it public, keep you updated.